INSTALLATION.md

```diff
@@ -1,10 +1,35 @@
-Installation
-============
+# Installation
 
-Create a GeoCAT-examples Conda environment
-------------------------------------------
-From the root directory of a cloned geocat-examples repository, run the following commands:
+This installation guide includes only the GeoCAT-examples installation instructions. 
+Please refer to [GeoCAT Contributor's Guide](https://geocat.ucar.edu/pages/contributing.html) for installation of 
+the whole GeoCAT project.
+  
+
+## Installing GeoCAT-examples
+
+GeoCAT-examples is not distributed as a conda package; thus, there is no conda installation for it.
+
+The easiest way to access GeoCAT-examples is cloning the repo and then using a 
+[Conda](http://conda.pydata.org/docs/) environment, building file of which is provided in this repo, as follows:
+
+### How to create a GeoCAT-examples Conda environment
+
+From the root directory of the cloned geocat-examples repository, run the following commands:
 ```
     conda env create -f conda_environment.yml -n geocat-examples
     conda activate geocat-examples
 ```
+
+ contains the dependencies for the required software packages, 
+such as ; therefore, there is no need for 
+explicitly installing those packages.
+
```

Comment:
Would it be worth mentioning that there are some cases when we need to add another package to the environment? I know this has only happened maybe a couple of times, so not sure if it is fully worth mentioning how someone may be able to add a new package if determined necessary.

Reply:
Agreed and addressed this in a new paragraph